components/hab/habitat/plan.ps1

```diff
@@ -19,12 +29,12 @@ function Invoke-Prepare {
     $env:SSL_CERT_FILE              = "$(Get-HabPackagePath "cacerts")/ssl/certs/cacert.pem"
     $env:LIB                        += ";$HAB_CACHE_SRC_PATH/$pkg_dirname/lib"
```

Comment:
This currently points to the lib directory in the `hab-win-deps` aws bundle. I found that if that bundle is missing the build fails.

Reply:
Actually, we can just delete lines 30 and 31. Oddly we need to rename libsodium.lib in tthe libsodium package to sodium.lib. Maybe there is some other variable to set to avoid that?